roda-ui/roda-wui/src/main/java/org/roda/wui/client/ingest/process/PluginParameterPanel.java

```diff
@@ -91,6 +93,8 @@ private void updateLayout() {
       createPluginSipToAipLayout();
     } else if (PluginParameterType.AIP_ID.equals(parameter.getType())) {
       createSelectAipLayout();
+    }else if (PluginParameterType.DISPOSAL.equals(parameter.getType())) {
```

Comment:
Having a plugin parameter of type DISPOSAL doesn't make much sense. What you want is to select AIPs that follow a certain filter. This filter should be configurable.